### ISSUE TITLE
sage-9.0: sagetex does NOT work at all

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#4432]
1. Make a tex file with this content (or really anything using sagetex):
```
\documentclass{article}
\usepackage{sagetex}
\begin{document}
\sageplot{plot(x^2, -1, 1)}
\end{document}
```
2. Hit the build button once.  
3. Look at the sagetex log and see this error:
```
Traceback (most recent call last):
  File "sage3.sagetex.sage.py", line 9, in <module>
    _st_ = sagetex.SageTeXProcessor('sage3', version='2019/01/09 v3.3', version_check=True)
  File "/ext/sage/sage-9.0/local/lib/python3.7/site-packages/sagetex.py", line 77, in __init__
    raise VersionError(errstr)
sagetex.VersionError: versions of .sty and .py files do not match.
sage3.sagetex.sage was generated by sagetex.sty version "2019/01/09 v3.3", but
is being processed by sagetex.py version "2019/11/14 v3.4".
Please make sure that TeX is using the sagetex.sty
from your current version of Sage; see
http://doc.sagemath.org/html/en/tutorial/sagetex.html.
```

**Comment by williamstein** [comment]
WORKAROUND: Type `sage_select 8.9` in a terminal.

**Comment by haraldschilly** [comment]
I've just had a quick idea: the sage select script could also set a symlink in ~/texmf pointing to the matching version of sage. I don't know which subdirectory exactly, but it shouldn't be too hard to figure out.

**Comment by williamstein** [comment]
Until this gets fixed, we should have a super-high-priority issue that is to check for `sagetex.VersionError` in the sagetex output, and if it is there, show an error, and say that you must use the default version of sage (e.g., delete ~/bin/sage).

**Comment by slel** [comment]
Note: many answers to SageTeX-related questions on the TeX LaTeX stack exchange
mention CoCalc as a convenient way to use SageTeX.

**Comment by williamstein** [comment]
> Note: many answers to SageTeX-related questions on the TeX LaTeX stack exchange
mention CoCalc as a convenient way to use SageTeX.

This ticket is now much, MUCH higher priority, because @haraldschilly recently switched cocalc to Ubuntu 20.04, and while he was at it -- unbeknownst to me (but it's a good idea) -- he made the default version of Sage be sage-9.1.  Thus sagetex would be now *broken by default*, if this ticket isn't fix.

Good news -- this ticket has been resolved; I just did a test and things seem fine with sage-9.1.
